src/EnergyPlus/ReportCoilSelection.cc

```diff
@@ -1233,24 +1249,28 @@ void ReportCoilSelection::setCoilCoolingCapacity(
             if (c->coilDesEntHumRat == -999.0) { // don't overwrite if already set directly
                 c->coilDesEntHumRat = DataSizing::FinalSysSizing(curSysNum).MixHumRatAtCoolPeak;
             }
-            c->coilDesEntWetBulb = Psychrometrics::PsyTwbFnTdbWPb(c->coilDesEntTemp, c->coilDesEntHumRat, DataEnvironment::StdBaroPress,
-                                                                  "ReportCoilSelection::setCoilCoolingCapacity");
+            c->coilDesEntWetBulb = Psychrometrics::PsyTwbFnTdbWPb(
+                c->coilDesEntTemp, c->coilDesEntHumRat, DataEnvironment::StdBaroPress, "ReportCoilSelection::setCoilCoolingCapacity");
             c->coilDesEntEnth = Psychrometrics::PsyHFnTdbW(c->coilDesEntTemp, c->coilDesEntHumRat);
             if (c->coilDesLvgTemp == -999.0) {
                 c->coilDesLvgTemp = DataSizing::FinalSysSizing(curSysNum).CoolSupTemp;
             }
             if (c->coilDesLvgHumRat == -999.0) { // don't overwrite if already set directly
                 c->coilDesLvgHumRat = DataSizing::FinalSysSizing(curSysNum).CoolSupHumRat;
             }
-            c->coilDesLvgWetBulb = Psychrometrics::PsyTwbFnTdbWPb(c->coilDesLvgTemp, c->coilDesLvgHumRat, DataEnvironment::StdBaroPress,
-                                                                  "ReportCoilSelection::setCoilCoolingCapacity");
+            c->coilDesLvgWetBulb = Psychrometrics::PsyTwbFnTdbWPb(
+                c->coilDesLvgTemp, c->coilDesLvgHumRat, DataEnvironment::StdBaroPress, "ReportCoilSelection::setCoilCoolingCapacity");
             c->coilDesLvgEnth = Psychrometrics::PsyHFnTdbW(c->coilDesLvgTemp, c->coilDesLvgHumRat);
             if (DataAirSystems::PrimaryAirSystem(curSysNum).NumOACoolCoils > 0) { // there is precooling of the OA stream
                 c->oaPretreated = true;
             }
         }
 
     } else if (curZoneEqNum > 0 && allocated(DataSizing::FinalZoneSizing)) {
+        c->zoneNum.resize(1);
+        c->zoneName.resize(1);
+        if (allocated(DataZoneEquipment::ZoneEquipConfig)) c->zoneNum[0] = DataZoneEquipment::ZoneEquipConfig(curZoneEqNum).ActualZoneNum;
+        if (allocated(DataZoneEquipment::ZoneEquipConfig)) c->zoneName[0] = DataZoneEquipment::ZoneEquipConfig(curZoneEqNum).ZoneName;
```

Comment:
First change added to allow zone name to be reported in Coil Sizing Details table.